x-pack/plugins/maps/public/classes/layers/solution_layers/observability/observability_layer_wizard.tsx

```diff
@@ -14,14 +14,20 @@ import { getIndexPatternService } from '../../../../kibana_services';
 
 export const ObservabilityLayerWizardConfig: LayerWizard = {
   categories: [LAYER_WIZARD_CATEGORY.ELASTICSEARCH, LAYER_WIZARD_CATEGORY.SOLUTIONS],
-  checkVisibility: async () => {
+  getIsDisabled: async () => {
     try {
       await getIndexPatternService().get(APM_INDEX_PATTERN_ID);
-      return true;
-    } catch (e) {
       return false;
+    } catch (e) {
+      return true;
     }
   },
+  disabledReason: i18n.translate('xpack.maps.observability.disabledDesc', {
+    defaultMessage: 'Cannot find APM index pattern {patternId}',
```

Comment:
Can you make the APM message more like the security message, something like "Cannot find APM index pattern. To get started with Observably, go to Observability > Overview."